corehq/apps/app_manager/xform.py

```diff
@@ -1504,6 +1508,8 @@ def _add_meta_2(self, form):
             '{orx}instanceID',
             '{cc}appVersion',
             '{orx}drift',
+            '{cc}formLoadTime',
+            '{cc}timeStartNumeric',
```

Comment:
I had a call with Ahmad and he mentioned its necessary to also add this field.

That said, it sounds like this work is on hold again, since there's not agreement on the mobile team on whether this should be done or not.